atoms/compareForm.ts

```diff
@@ -0,0 +1,52 @@
+import { atom } from 'recoil'
```

Comment:
A couple of comments from trying the feature:

1. when I click on the compare icon the modal opens with a strange lateral shift effect;
2. I'm not sure why Text only or Image Only should be that big and within a rectangle; These could simply be small icons with text
3. Image / Text Only -> the Only part doesn't sound great. It could be Select poll type: Text - Image;
4. After clicking on Text only or Image only: the "< Text Only" or "< Image Only" should be "< Back" and clicking on "Back" should also bring the user back (not only clicking on the "<" icon;
5. In < Text/Image Only there is no need for a tooltip "Go Back";
6. On Text Only: Label your first/second option should simply be "First option ..." and "Second option..." without the title First Option and Second Option and without the placeholders Label your first... it is not a label, it is the option;
7. On Image Only, I would remove First Option and Second Option. It seems clear. Also Upload your image should take as much space as the label on the screen. Also, I would replace "Label your first/second option" with "Caption (optional)";
8. In general, these multiple choices and constraints on text/image only just make the app hard to use and requires thinking for something that should be as quick as writing a question on whatsapp. I honestly don't like this initial forced choice between text or image only. Makes me stress just to see that initial option. Then I have another screen and now I need to upload 2 images and maybe give a label. So much effort. I cannot paste a picture here but look how easy it is on Reddit to create a new post (using tabs to switch between text or images and video).

Reply:
Other errors:

1. If I first click on Compare icon and then on Upload Image icon the compare options don't disappear. I understand that there could be an image + poll but then we should have a button to cancel the Compare option if the user clicked on the icon by mistake;
2. Compare should be Poll probably or better "Create poll";
3. I would remove the "Select Compare Type" and probably just have tabs like creating a post on Reddit or a title "Poll" and under some text/image icons like Twitter.